### PR TITLE
Improving "--help" text ("--data" description is better now)

### LINKED ISSUE #77: Improve help command description
Help command displays less information than it is necessary to run ```# ttsnake --data``` command. It shows that we need to run ```# ttsnake --data```, but after running it, we receive an output saying that we must provide an option argument to ```# ttsnake --data```. This additional option should be the non-default path to move the data. 
```
ttsnake --data
ttsnake: option '--data' requires an argument
Usage: ttsnake [options]

  Options

  -h, --help       Display this information message.
  -d, --data       Selects a non-default data path
  -v, --version    Outputs the program version
  ```

### PR DESCRIPTION
**Pull request by Haltz01**
Fixes #77 .

This PR proposes the following changes:
- Improving the "--help" text. "--data" description continues simple, but more descriptive;
- Changing order of [options] avaliable to run ttsnake (alphabetical order).

Very simple changes, but even I didn't understand at first glance what "--data" was used for.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,8 +153,8 @@ If, for some reason, the command returns a message asking for a minimum terminal
          Options
 
          -h, --help      Displays this information message
-         -d, --data      Selects a non-default data path
-	 -v, --version   Outputs the program version
+         -d, --data      Selects a non-default data path to game's assets and "share" folder
+	 	 -v, --version   Outputs the program version
```
Comment (math-araujo) [inline README.md]: I would suggest to keep `-v, --version` aligned with the other options on README. What do you think?

Comment (Haltz01) [reply README.md]: You're talking about the spacing (number of tabs/spaces)? 
For me it was aligned , but I can double check and fix it if that's what you're talking about!

Comment (Haltz01) [reply README.md]: I believe it's fixed now! Can you confirm @math-araujo ?

Comment (math-araujo) [reply README.md]: Yeah, it's fixed now! Thanks :)

Comment (Haltz01) [reply README.md]: Nice :)
I can't review the PR so if you want to merge this request for us, it'd be great

**Review by math-araujo — APPROVED** [review]
LGTM :)

**Review by gabriel-libardi — APPROVED** [review]

**Pull request merged**